sdcm/scan_operation_thread.py

```diff
@@ -191,6 +193,18 @@ def fetch_result_pages(self, result, read_pages):
             if read_pages > 0:
                 pages += 1
 
+    @contextmanager
+    def cql_connection(self, **kwargs):
+        node = kwargs.get("node", self.db_node)
```

Comment:
you need to pop out the node from kwargs, if you want to specify it later
or use setdefault 

the unittest demonstrate it's currently broken